docs/links.md

```diff
@@ -9,5 +9,58 @@ You can configure Argo Server to show custom links:
 * A "Get Help" button in the bottom right of the window linking to you organisation help pages or chat room. 
 * Deep-links to your facilities (e.g. logging facility) in the user interface for both the workflow and each workflow pod.
 
-See [workflow-controller-configmap.yaml](workflow-controller-configmap.yaml) 
+Links can contain placeholder variables. Placeholder variables are indicated by the dollar sign and curly braces (`${variable}`).
+There are currently 4 variables available:
 
+| Variable                | Description                                                                                                                                               |
+|:-----------------------:|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `${metadata.namespace}` | Kubernetes namespace of the current workflow/pod/event source/sensor                                                                                      |
+| `${metadata.name}`      | Name of the current workflow/pod/event source/sensor                                                                                                      |
+| `${status.startedAt}`   | Start timestamp of the workflow/pod, in the format of `2021-01-01T10:35:56Z`                                                                              |
+| `${status.finishedAt}`  | End timestamp of the workflow/pod, in the format of  `2021-01-01T10:35:56Z`. If the workflow/pod is still running, this variable will be an empty string. |
+
+> v3.1 and after
```

Comment:
I don't see 3.1 branch created yet so I assume this change will go in 3.1 release (if merged)?